rust/src/ftp/response.rs

```diff
@@ -22,11 +22,12 @@ use std::slice;
 
 #[repr(C)]
 pub struct FTPResponseLine {
-    code: *mut u8,     // Response code as a string (may be null)
-    response: *mut u8, // Response string
-    length: usize,         // Length of the response string
-    truncated: bool,       // Uses TX/state value.
-    total_size: usize,     // Total allocated size in bytes
+    code: *mut u8,      // Response code as a string (may be null)
+    response: *mut u8,  // Response string
+    length: usize,      // Length of the response string
+    code_length: usize, // Length of the response string
```

Comment:
Is the comment still correct? Or is `code_length` just the length of the code (ie: "150")?

Reply:
Thanks - this field stores the length of the code -- "150" -- in your example. Since the size is static, storing the length removes a log-time call to `strlen`